modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,219 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
+  # When used against a cluster ensure the service being used is pointing to all the cluster instances.
+  # This could be the loadbalancer from neo4j helm chart or the headless service installed via neo4j-headless-service helm chart
+  serviceName: ""
+  # default is set to cluster.local
+  domain: "cluster.local"
+
+  # This assumes ingress-nginx controller or haproxy-ingress-controller is already installed in your kubernetes cluster.
+  # You can install ingress-nginx by following instructions on this link https://github.com/kubernetes/ingress-nginx/blob/main/docs/deploy/index.md#quick-start
+  # You can install haproxy-ingress by following instructions on this link https://haproxy-ingress.github.io/docs/getting-started/
+  ingress:
+    enabled: true
+    #default value is nginx. It can be either nginx or haproxy
+    className: nginx
+    annotations: {}
+#      "demo": "value"
+#      "demo2": "value2"
+    tls:
+      enabled: false
+      config: []
+#      - secretName: "demo2"
+#        hosts:
+#          - localhost
+----
+
+==== Install the Helm chart
+
+The following steps assume that you have a Kubernetes cluster running and a standalone Neo4j helm chart installed.
+The standalone Neo4j has a Neo4j service with the name _standalone-admin_, and it has _:7474_ an _:7687_ opened.
+To verify that, run `get all, pvc, pv, configmaps, secrets`.
+
+. You need to have an Ingress controller for the Kubernetes Ingress to work.
+You can use the following command to install the Nginx Ingress controller:
++
+[source,shell]
+----
+helm upgrade --install ingress-nginx ingress-nginx \
+      --repo https://kubernetes.github.io/ingress-nginx \
+      --namespace ingress-nginx --create-namespace
+----
+
+. If your Neo4j is running on _:443_, then set the `tls.enabled` to `true` and create a secret with the following command:
+.. Create Ingress certificates:
++
+[source,shell]
+----
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ingress.key -out ingress.cert -subj "/CN=localhost/O=neo4j" -addext "subjectAltName = DNS:localhost"
+----
+.. Create Kubernetes secret using the Ingress certificates:
++
+[source,shell]
+----
+kubectl create secret tls ingress-cert --key /path/to/your/certs/ingress.key --cert /path/to/your/certs/ingress.cert
+----
+
+. Configure the _ingress.yaml_ file with the correct values for the `serviceName` and `secretName`.
+Ensure that the `secretName` is the same as the one created in the previous step.
+Enable TLS by setting `tls.enabled` to `true`.
++
+[source, yaml]
+----
+reverseProxy:
+    image: neo4j/neo4j-reverse-proxy:5.12.0
+    serviceName: "standalone-admin"
+    ingress:
+        enabled: true
+        tls:
+            enabled: true
+            config:
+                - secretName: ingress-cert
+                  hosts:
+                    - localhost
+----
+
+. If your Neo4j is running on _:80_, then set the `tls.enabled` to `false` and create the _ingress.yaml_ file with the following content:
```

Comment:
Neo4j is never running on 80 or 443 its the ingress...so we can say if you "want" to access Neo4j on port 80